src/renderers/shared/fiber/ReactFiberBeginWork.js

```diff
@@ -50,7 +53,28 @@ function updateHostComponent(current, workInProgress) {
   console.log('host component', workInProgress.type, typeof workInProgress.pendingProps.children === 'string' ? workInProgress.pendingProps.children : '');
 
   var nextChildren = workInProgress.pendingProps.children;
-  reconcileChildren(current, workInProgress, nextChildren);
+
+  let priority = workInProgress.pendingWorkPriority;
+  if (workInProgress.pendingProps.hidden && priority !== OffscreenPriority) {
```

Comment:
Are these props the same as the DOM props we have now? This lookup might be slow because of all the different shapes. Not sure if that is a concern but everything else I've seen so far looks monomorphic.